modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/DeploymentConfig.java

```diff
@@ -25,8 +25,10 @@
     private Integer replicas;
     private Map<String, String> labels;
     private List<Container> containers;
+    private Container container;
     private String servicePort;
     private Integer proxyPort;
+    private List secrets;
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/7d70ba5f5bbacfe7789db67072db49c2ff893b78